resources/zib/zib-Payer.xml

```diff
@@ -0,0 +1,163 @@
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-payer-v3.1.1-2020EN" />
+    <uri value="https://zibs.nl/wiki/Payer-v3.1.1(2020EN)" />
+    <name value="zib Payer-v3.1.1(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Coverage" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Coverage" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Coverage">
+      <path value="Coverage" />
+      <short value="Payer / Insurance" />
+      <comment value="The zib Payer is mapped to this Coverage profile and a profile on Organization (&amp;lt;http://nictiz.nl/fhir/StructureDefinition/zib-Payer-Organization&amp;gt;). This Coverage profile acts as the focal resource of Payer, although Coverage is more associated with the Insurance container of the zib then with Payer as a whole. Nevertheless, the Coverage resource acts as an overarching resource that references the various resources that represent PayerPerson or InsuranceCompany and adds details like BankInformation. For the InsuranceCompany concept, the profile on Organization is used which is referenced using the `payor` element." />
+      <alias value="Betaler" />
+      <alias value="Verzekering" />
+      <mapping>
+        <identity value="zib-payer-v3.1.1-2020EN" />
+        <map value="NL-CM:1.1.1" />
+        <comment value="Payer" />
+      </mapping>
+      <mapping>
+        <identity value="zib-payer-v3.1.1-2020EN" />
+        <map value="NL-CM:1.1.8" />
+        <comment value="Insurance" />
+      </mapping>
+    </element>
+    <element id="Coverage.status">
+      <path value="Coverage.status" />
+      <comment value="Sending systems that don't record an explicit status can use the following guidance to infer a value from the zib:&#xD;&#xA;&#xD;&#xA;* EndDateTime present:&#xD;&#xA;    * and in the future: _active_&#xD;&#xA;    * and in the past: _cancelled_&#xD;&#xA;* When no status can be derived from the StartDateTime and EndDateTime, e.g. the Coverage concerns a PayerPerson, it is expected that only _active_ Coverage resources are exchanged." />
+      <mapping>
+        <identity value="zib-payer-v3.1.1-2020EN" />
+        <map value="NL-CM:1.1.14" />
+        <comment value="EndDateTime (implicit, main mapping is on Coverage.period.start)" />
+      </mapping>
+    </element>
+    <element id="Coverage.type">
+      <path value="Coverage.type" />
+      <short value="InsuranceType" />
+      <definition value="Type of insurance policy. Codes as returned in the Check for Right to Insurance" />
+      <alias value="Verzekeringssoort" />
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.1.1.1--20200901000000" />

```

Comment:
Because of the required binding, the `.type` cannot be used to indicate a self-pay which may be appropriate if the PayerPerson is exchanged. Should this become an extensible binding?

Reply:
FIXED by splitting the profile into two based on the two containers InsuranceCompany and PayerPerson